src/cdef_apply.rs

```diff
@@ -173,6 +173,12 @@ pub(crate) fn rav1d_cdef_brow<BD: BitDepth>(
     let uv_stride: ptrdiff_t = BD::pxstride(f.cur.stride[1]);
 
     let mut bit = false;
+
+    // In C, this is declared uninitialized inside the loop,
+    // so it doesn't matter what it is initialized in Rust as long as it is initialized,
```

Comment:
```suggestion
    // so it doesn't matter what it is initialized to in Rust as long as it is initialized,
```

Sorry, I made a typo.

Reply:
Fixed 😄